src/elements/content-explorer/ContentExplorer.js

```diff
@@ -290,6 +296,13 @@ class ContentExplorer extends Component<Props, State> {
         } else {
             this.fetchFolder(currentFolderId);
         }
+
+        if (isFeatureEnabled(features, 'contentExplorer.gridView.enabled')) {
+            const pref = this.store.getItem(localStoreViewMode);
+            if (pref && (pref === VIEW_MODE_GRID || pref === VIEW_MODE_LIST)) {
```

Comment:
Initial `pref` check is to pass `flow`.  Also, not sure if it's possible to dynamically check if `pref` is of type `ViewMode`, so had to do the two equality checks.

Reply:
Is the initial `pref` identity check needed? Seems like the two equality checks should be enough, since null/undefined won't result in `true`.